docs/versioned_docs/version-v0.5/engines/acp-ory.md

```diff
@@ -498,6 +498,54 @@ The following access request would be denied.
 }
 ```
 
+### Time Interval Condition
```

Comment:
Can you please add these changes to docs/docs/engines/acp-ory.md as well? Thank you!

Reply:
Yeah no problem.